src/modules/commitment/tests_impl.h

```diff
@@ -133,6 +134,10 @@ static void test_commitment_api(void) {
     CHECK(secp256k1_pedersen_commitment_serialize(sign, out2, &commit2) == 1);
     CHECK(memcmp(out, out2, 33) == 0);
 
+    /* Test conversion of commit to pubkey and back */
+    CHECK(secp256k1_pedersen_commitment_to_pubkey(sign, &pubkey, &commit) == 1);
+    CHECK(secp256k1_pubkey_to_pedersen_commitment(sign, &commit, &pubkey) == 1);
```

Comment:
Can you store the output of `secp256k1_pubkey_to_pedersen_commitment` in a new variable and `memcmp` it with `commit`?